source/reference/editor-mode.txt

```diff
@@ -246,10 +246,19 @@ Using Visual Studio as an External Editor
 Visual Studio requires a special parameter to work as an external
 editor. Use ``--wait`` with Visual Studio.
 
+Set an environment variable:
+
 .. code-block:: javascript
 
    export EDITOR="/usr/local/bin/code --wait"
 
+Set the Visual Studio editor with
+:ref:`config.set() <mongosh-shell-settings>`:
+
+.. code-block:: javascript
+
+   config.set("editor", "code --wait")
```

Comment:
Hey Dave, this looks good and in my experience worked much better than the absolute path.

Only thing I would add is a note that specifies that "code" must be specified as an alias or configured with your terminal for "code" to work. I remember "code ." did not work straight out of the box for me, something like :  

.. note::
   
   If using the code alias visual studio code needs to be configured as an alias for your terminal or command line   environment.

Reply:
Done.

Reply:
LGTM!